client/src/app/+videos/+video-watch/video-watch.component.ts

```diff
@@ -307,6 +307,12 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
     )
   }
 
+  isChannelDisplayNameGeneric () {
+    const genericChannelDisplayName = `Main ${this.video.channel.ownerAccount.name} channel`
```

Comment:
At the beginning of peertube it was also `Default ... channel`. Please add this condition too (or use an array)